src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogQueuePublisher.java

```diff
@@ -96,6 +131,13 @@ protected void doRun() throws Exception {
         } catch (Exception e) {
             DatadogUtilities.severe(logger, e, null);
         }
-
+    }
+    
+    public DatadogClient getDatadogClient(){
```

Comment:
If I use the testing strategy [here](https://github.com/jenkinsci/datadog-plugin/pull/74/files#diff-5f1692b010bca325cd855cb018b01733L33), I won't need this.